frontend/packages/pipelines-plugin/src/components/repository/RepositoryPipelineRunRow.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { TableRow, TableData, RowFunction } from '@console/internal/components/factory';
+import { ResourceLink, Timestamp, truncateMiddle } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunModel } from '../../models';
+import { PipelineRunKind } from '../../types';
+import { getPipelineRunKebabActions } from '../../utils/pipeline-actions';
+import { pipelineRunFilterReducer } from '../../utils/pipeline-filter-reducer';
+import { pipelineRunDuration } from '../../utils/pipeline-utils';
+import LinkedPipelineRunTaskStatus from '../pipelineruns/status/LinkedPipelineRunTaskStatus';
+import PipelineRunStatus from '../pipelineruns/status/PipelineRunStatus';
+import { ResourceKebabWithUserLabel } from '../pipelineruns/triggered-by';
+import { RepositoryLabels, RepositoryFields } from './consts';
+import { tableColumnClasses } from './RepositoryPipelineRunHeader';
+
+const pipelinerunReference = referenceForModel(PipelineRunModel);
+
+type PLRStatusProps = {
+  obj: PipelineRunKind;
+};
+
+const PLRStatus: React.FC<PLRStatusProps> = ({ obj }) => {
+  return (
+    <PipelineRunStatus
+      status={pipelineRunFilterReducer(obj)}
+      title={pipelineRunFilterReducer(obj)}
+      pipelineRun={obj}
+    />
+  );
+};
+
+const PipelineRunRow: RowFunction<PipelineRunKind> = ({ obj, index, key, style }) => {
+  const plrLabels = obj.metadata.labels;
+
+  return (
+    <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink
+          kind={pipelinerunReference}
+          name={obj.metadata.name}
+          namespace={obj.metadata.namespace}
+          data-test-id={obj.metadata.name}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[1]} columnID="sha">
+        <Tooltip content={<>{plrLabels[RepositoryLabels[RepositoryFields.SHA]]}</>}>
+          <div className="co-truncate co-nowrap">
+            {truncateMiddle(plrLabels[RepositoryLabels[RepositoryFields.SHA]], {
+              length: 10,
+              truncateEnd: true,
+            })}
+          </div>
+        </Tooltip>
```

Comment:
Tooltip here should show the commit message not the `sha`,  commit message for a pipelinerun is now available in the annotations. 
eg
annotations: {
  pipelinesascode.tekton.dev/sha-title: "My commit message"
}
Here is the PR for this commit message - https://github.com/openshift-pipelines/pipelines-as-code/issues/193

Reply:
updated